### DISCUSSION TITLE
Welcome to anibot Discussions!

### DISCUSSION THREAD

**Discussion by lostb053** [lostb053/anibot#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by icenasser** [comment]
Is there a way for a bot to recurre daily message?

**Reply by lostb053** [reply]
Can you explain a bit more? I don't completely get your idea.
Tho, you can always code tbh. I doubt i made a feature fitting the detail you gave.
